roles/artifacts/tasks/environment.yml

```diff
@@ -24,3 +24,29 @@
       ip ro ls > {{ cifmw_artifacts_basedir }}/artifacts/ip-network.txt
       ip rule ls >> {{ cifmw_artifacts_basedir }}/artifacts/ip-network.txt
       ip -j -p link ls >> {{ cifmw_artifacts_basedir }}/artifacts/ip-network.txt
+
+- name: Find yum repos files
+  become: true
+  ansible.builtin.find:
+    paths: "/etc/yum.repos.d/"
+    patterns: "*.repo"
+    recurse: false
+  register: _repo_files
+
+- name: Create repos directory
+  ansible.builtin.file:
+    path: "{{ cifmw_artifacts_basedir }}/logs/yum_repos"
+    state: directory
+    mode: "0777"
```

Comment:
I don't we need to set 777. It will work even without this, mainly because the user and group are set just fine.